model/src/ww3_outp.F90

```diff
@@ -361,31 +371,63 @@ PROGRAM W3OUTP
   !
   !--- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
   ! 3.  Read general data and first fields from file
+  !     Output time, time step, number of steps, optional dynpnt and prefix
   !
+  CALL NEXTLN ( COMSTR , NDSI , NDSE )
+  WORDS = ''
+  READ (NDSI, '(A)', IOSTAT=IERR, END=801, ERR=802) LINEIN
+  READ(LINEIN,*,IOSTAT=IERR) WORDS
+  READ(WORDS(1), *, IOSTAT=IERR) TOUT(1)  ! Date (yyyymmdd)
+  READ(WORDS(2), *, IOSTAT=IERR) TOUT(2)  ! Time (hhmmss)
+  READ(WORDS(3), *, IOSTAT=IERR) DTREQ
+  READ(WORDS(4), *, IOSTAT=IERR) NOUT
+  IF (WORDS(5) /= '') READ(WORDS(5), *, IOSTAT=IERR) dynpnt
```

Comment:
These two additional new optional variables should be described and included in https://github.com/NOAA-EMC/WW3/blob/develop/model/inp/ww3_outp.inp

Reply:
Thanks for adding this.

Reply:
Please add an issue for the "to do" of reading in these variables from nml files or add this capability to this PR.